Dockerfile

```diff
@@ -9,9 +9,10 @@ RUN apt-get update && apt-get install -y \
     gdal-bin \
     libproj-dev \
     libgeos-dev \
-    libspatialite-dev
+    libspatialite-dev \
+    git
 RUN pip3 install -U pip setuptools wheel
-RUN pip3 install -U .
+RUN git clone https://github.com/openwisp/openwisp-controller && cd openwisp-controller/ && pip3 install -U .
 RUN echo "openwisp-controller installed"
 WORKDIR tests/
 CMD ["./docker-entrypoint.sh"]
```

Comment:
I am not sure if It will work without something like to change the path of spatialite according to `stretch`:
```
echo "SPATIALITE_LIBRARY_PATH = 'mod_spatialite.so'" >> settings.py
```
have you tested it?

Reply:
Yes, i had tested it. Docker image will built fine without any error with these changes.

![Screenshot from 2019-06-03 11-20-55](https://user-images.githubusercontent.com/20985005/58779062-566bdc80-85f2-11e9-8f1c-a3d1c5b1f40a.png)
![Screenshot from 2019-06-03 11-23-45](https://user-images.githubusercontent.com/20985005/58779070-5966cd00-85f2-11e9-9833-5923f434df08.png)
![Screenshot from 2019-06-03 11-23-49](https://user-images.githubusercontent.com/20985005/58779074-5c61bd80-85f2-11e9-8645-7098df53d694.png)

Reply:
`spatialite` errors will pop up when you runserver. 
Try: `docker run openwisp/controller:latest`